origami.json

```diff
@@ -2,7 +2,7 @@
 	"supportStatus": "deprecated",
 	"origamiType": "module",
 	"description": "A component used to test origami tools and services",
```

Comment:
I wonder if we should move `description` to `package.json` since it won't be generated anymore, and we probably want to show a description on npm? Maybe not in this spec version to keep the scope down. Could be in a future spec major, where we remove other duplicaton between `origami.json` and `package.json` too.

Reply:
That's a good idea :-)

Reply:
I've added a description to the package.json which is the same as the one in origami.json